test/e2e/azure_test.go

```diff
@@ -350,4 +350,99 @@ var _ = Describe("Workload cluster creation", func() {
 			})
 		})
 	})
+
+	Context("Creating a Windows Enabled cluster", func() {
```

Comment:
Same question about merging tests here. Are we adding more test coverage by testing a cluster _without_ Windows nodes?

Reply:
This is due to flannel being the currently supported cni.  We have split the test jobs for now and as calico support comes online will re-address this